config/kernel/linux-rockchip64-current.config

```diff
@@ -4574,8 +4574,8 @@ CONFIG_MFD_AAT2870_CORE=y
 # CONFIG_MFD_BCM590XX is not set
 # CONFIG_MFD_BD9571MWV is not set
 CONFIG_MFD_AC100=m
-CONFIG_MFD_AXP20X=m
-# CONFIG_MFD_AXP20X_I2C is not set
+CONFIG_MFD_AXP20X=y
```

Comment:
Is there a reason  why this and `CONFIG_REGULATOR_AXP20X` cannot be built as modules?

Reply:
oops sorry I didn't notice before, lemme fix it

Reply:
it reference by [linux-rockchip64-current.config#L4408](https://github.com/clockworkpi/DevTerm/blob/e47841188f1b0ddd7e1a5aefb58dd5c7dd1754ed/Code/patch/armbian_build_a06/patch/linux-rockchip64-current.config#L4408) and I think it work as MFD device

Reply:
axp is allwinner thing, so IMO not relevant here.

Reply:
they use axp22x for control Li-ion battery / DC power supply
![image](https://user-images.githubusercontent.com/16154023/167290390-89b20dfe-2da0-4a3e-b45c-d5348ad3f02b.png)

Reply:
aha, ok. then legit